### PR TITLE
Add get all customers

### PR DESCRIPTION
**Pull request by kerrermanisNL**
Add a `get_all_customers` function. Also refactored some stuff to deduplicate code.

### CODE REVIEW

**Comment by kerrermanisNL** [comment]
Note that I didn't refactor the `get_app*` tests because they were relatively annoying to test in some other way. I also think it's fine like it is now.

File: tests/client/test_client.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import requests
+
+from tests.test import TestBase
+
+
+class TestClient(TestBase):
+    def test_sets_api_url_to_provided_site(self):
+        self.assertEqual(self.chargebee_client.api_url, 'https://my_site.chargebee.com/api/v2')
+
+    def test_sets_auth_to_hhtp_basic_auth_method_with_api_key_as_username(self):
```
Comment (dmeijboom) [inline tests/client/test_client.py]: * `http*

**Review by dmeijboom — APPROVED** [review]
Apart from one typo: LGTM :+1:

File: tests/client/test_client.py (hunk 1 above)
Comment (kerrermanisNL) [reply tests/client/test_client.py]: Fixed

**Pull request merged**